cli/cmd/network.go

```diff
@@ -0,0 +1,171 @@
+package cmd
```

Comment:
Could we add a (sub)command to `network` command or a new command that would show you the current default network and/or the current default ParaTime?

Or perhaps the default network could be shown via a `(default)` suffix (or similar) when running `oasis network list`?